README.md

```diff
@@ -13,11 +13,27 @@ The GenAIInfra repository is organized under four main directories, which includ
 | `manifests`         | Manifests for deploying [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes and on Docker Compose. |
 | `scripts`           | Scripts for testing, tools for facilitate OPEA deployment, and etc.                                                         |
 
-## Usages
+## Prerequisite
+
+GenAIInfra uses Kubernetes as the cloud native infrastructure. Please follow the following steps to prepare the Kubernetes environment.
+
+### Setup Kubernetes cluster
+
```

Comment:
Suggest to use or add the kubespray link, which is used for production ready k8s deployment with simple and clear instructions.
https://github.com/kubernetes-sigs/kubespray/blob/master/README.md

Reply:
done